src/pages/tag.js

```diff
@@ -95,6 +96,12 @@ export default class TagGridPage extends Component {
           meta={[{ name: "description", content: shop && shop.description }]}
         />
         <Breadcrumbs isTagGrid={true} tag={tag} tags={tags} />
+        {
+          tag && tag.displayTitle ? (
```

Comment:
I think `{tag && tag.displayTitle && <ProductGridTitle displayTitle={tag.displayTitle} />}` is a little more concise  rather than the full ternary operator in jsx.

Reply:
Resolved